src/cbmc/cbmc_solvers.cpp

```diff
@@ -189,8 +187,6 @@ class cbmc_solver_with_filet:public cbmc_solverst::solvert
   ~cbmc_solver_with_filet()
   {
     // delete the prop before the file
-    delete prop_conv_ptr;
```

Comment:
I suspect this is still needed, as otherwise prop has a ref to a deleted object. The assertion virtual ~solvert() needs to go away.